demos/README.md

```diff
@@ -0,0 +1,38 @@
+# FlexModel Demos
+**Note**: If your system does not have InfiniBand, please be sure to run `export NCCL_IB_DISABLE=1` prior to running the experiments.
+## Induction Heads
+
+The file `induction_heads_multigpu.py` takes in a Llama-2 HuggingFace model path and outputs two plots which visualize the per token loss and a heatmap of the induction scores across all attention heads.
+
+Before running the file, you might need to insert `DummyModule` from `flex_model.core` in order to store the activation maps. `DummyModule` is just an `nn.Module` that is an identity function and its primary purpose is to allow us to attach a `HookFunction` at any arbitrary point during a forward pass. An example of this is below
```

Comment:
minor gripe: "An example of this is [shown] below[:]". Don't need to have "
### Llama-2 Attention Module" sub-header.